src/client/api/mgmt.c

```diff
@@ -30,27 +30,6 @@
 #include "client_internal.h"
 #include "task_internal.h"
 
-int
```

Comment:
I found this external hdf5 vol-daos usage of daos_mgmt_svc_rip(). @mchaarawi will its removal here cause a testing impact?
https://github.com/HDFGroup/vol-daos/blob/0a51e197b8ce6675b3b8a0bada87edae334aa8e4/test/daos_vol/h5daos_test_recovery.c#L118

Reply:
Great finding. Checked with Mohamad, agreed we should move vol-daos code to use dmg before removing daos_mgmt_svc_rip(). Will create a new ticket for vol-daos to make the change, but that need to be landed before we can land this one to avoid disruption to vol-daos.

Reply:
The macsio failure is related to this vol-daos issue:

11:05:17 DEBUG| [stderr] [1,0]<stderr>:[wolf-112:124159:0:124159] Caught signal 11 (Segmentation fault: address not mapped to object at address 0x12)
11:05:17 DEBUG| [stderr] [1,0]<stderr>:==== backtrace ====
11:05:17 DEBUG| [stderr] [1,0]<stderr>:    0  /lib64/libucs.so.0(+0x17970) [0x7f5f3c85c970]
11:05:17 DEBUG| [stderr] [1,0]<stderr>:    1  /lib64/libucs.so.0(+0x17b22) [0x7f5f3c85cb22]
11:05:17 DEBUG| [stderr] [1,0]<stderr>:    2  /lib64/libc.so.6(+0x36400) [0x7f5f4f406400]
11:05:17 DEBUG| [stderr] [1,0]<stderr>:    3  /lib64/libdaos.so.0(+0x31a1a) [0x7f5f15d1da1a]
11:05:17 DEBUG| [stderr] [1,0]<stderr>:    4  /lib64/libdaos_common.so(+0x42a6d) [0x7f5f15ab0a6d]
11:05:17 DEBUG| [stderr] [1,0]<stderr>:    5  /lib64/libdaos.so.0(daos_progress+0x29) [0x7f5f15d191c9]
11:05:17 DEBUG| [stderr] [1,0]<stderr>:    6  /usr/lib64/openmpi3/lib/libhdf5_vol_daos.so.1(+0x177d6) [0x7f5f160257d6]
11:05:17 DEBUG| [stderr] [1,0]<stderr>:    7  /usr/lib64/openmpi3/lib/libhdf5_vol_daos.so.1(+0x65bcc) [0x7f5f16073bcc]
11:05:17 DEBUG| [stderr] [1,0]<stderr>:    8  /usr/lib64/openmpi3/lib/libhdf5.so.201(+0x40b9bd) [0x7f5f503c89bd]
11:05:17 DEBUG| [stderr] [1,0]<stderr>:    9  /usr/lib64/openmpi3/lib/libhdf5.so.201(H5VL_file_create+0x9e) [0x7f5f503d3aee]
11:05:17 DEBUG| [stderr] [1,0]<stderr>:   10  /usr/lib64/openmpi3/lib/libhdf5.so.201(H5Fcreate+0x26a) [0x7f5f5011eb0a]
11:05:17 DEBUG| [stderr] [1,0]<stderr>:   11  /usr/lib64/openmpi3/bin/macsio() [0x41c207]
11:05:17 DEBUG| [stderr] [1,0]<stderr>:   12  /usr/lib64/openmpi3/bin/macsio() [0x41a92f]
11:05:17 DEBUG| [stderr] [1,0]<stderr>:   13  /usr/lib64/openmpi3/bin/macsio(main+0x82f) [0x40d4af]
11:05:17 DEBUG| [stderr] [1,0]<stderr>:   14  /lib64/libc.so.6(__libc_start_main+0xf5) [0x7f5f4f3f2555]
11:05:17 DEBUG| [stderr] [1,0]<stderr>:   15  /usr/lib64/openmpi3/bin/macsio() [0x40dc60]
11:05:17 DEBUG| [stderr] [1,0]<stderr>:===================

Reply:
vol_daos tested with daos_mgmt_svc_rip() replaced with dmg cmd and integrated to its master branch.